config-controller/pkg/client/client.go

```diff
@@ -0,0 +1,301 @@
+package client
+
+//go:generate mockgen-wrapper CachedPolicyClient,PolicyClient
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	storage "github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/clientconn"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/mtls"
+	"github.com/stackrox/rox/pkg/retry"
+	"github.com/stackrox/rox/pkg/size"
+	"google.golang.org/grpc"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var centralHostPort = fmt.Sprintf("central.%s.svc:443", env.Namespace.Setting())
```

Comment:
Comment from @clickboo from https://github.com/stackrox/stackrox/pull/12630: There could be an env var to read for this instead of hard coding this. Wary of the hard coding.

Reply:
We generally have this as an env var in the deployment spec for the config controller (and the helm charts already have a precedent I believe)

Reply:
After a bit of digging, I found [ROX_ENDPOINT](https://github.com/stackrox/stackrox/blob/d78c62fd4d8ee96ae0bb513f0aaaa4db830dd096/pkg/env/roxctl.go#L14), but it's only used in roxctl code; I don't see it in deployment templates.